### PR TITLE
Fixes Issue 18: Sanitize HTML from feed posts (XSS, script, etc removal)

### LINKED ISSUE #18: Sanitize HTML from feed posts (XSS, script, etc removal)
Sanitizer should clean any HTML code found in the blog post to eliminate possibility of running some else's script in our system.

### PR DESCRIPTION
**Pull request by shmooey**
Fixes #18

### CODE REVIEW

**Comment by miggs125** [comment]
@shmooey friendly reminder to add a test for all code that you push :)

**Comment by shmooey** [comment]
Thanks for the feedback so far! I'll make the changes

**Comment by shmooey** [comment]
Ok, I've left JSDOM in for now and added a test. How does this look? @humphd @manekenpix @miggs125

File: src/sanitize-html.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Sanitize HTML and prevent XSS attacks using DOMPurify
+// eg. DOMPurify.sanitize('<img src=x onerror=alert(1)//>'); // becomes <img src="x">
+
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
+
+module.exports.run = async function(text) {
+  const { window } = new JSDOM('');
+  const DOMPurify = createDOMPurify(window);
+  return Promise.resolve(DOMPurify.sanitize(text));
```
Comment (manekenpix) [inline src/sanitize-html.js]: I'm curious about this, why wrapping it in a promise? I checked the [dompurify repo](https://github.com/cure53/DOMPurify) and it doesn't seem to need that.

Comment (humphd) [reply src/sanitize-html.js]: Some DOM implementations will be sync, others async.  If we wrap this in a Promise based API, we can choose either or, without changing all the call sites.

File: test/sanitize-html.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Test for sanitize html
+
+const sanitizeHTML = require('../src/sanitize-html');
+
+test('Tests for sanitized HTML', async () => {
+  const data = await sanitizeHTML.run('<img src=x onerror=alert(1)//>');
```
Comment (humphd) [inline test/sanitize-html.test.js]: Drop the `//` from the end of the tag.

**Comment by shmooey** [comment]
Changes made!

**Pull request merged**